sql/firefox_accounts/fxa_users_services_last_seen/view.sql

```diff
@@ -2,13 +2,10 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.fxa_users_services_last_seen`
 AS
 SELECT
-  -- We cannot use UDFs in a view, so we paste the body of udf.bitpos(bits) literally here.
-  CAST(SAFE.LOG(days_seen_bits & -days_seen_bits, 2) AS INT64) AS days_since_seen,
-  CAST(
-    SAFE.LOG(days_seen_in_tier1_country_bits & -days_seen_in_tier1_country_bits, 2) AS INT64
-  ) AS days_since_seen_in_tier1_country,
-  CAST(SAFE.LOG(days_registered_bits & -days_registered_bits, 2) AS INT64) AS days_since_registered,
-  * REPLACE (cn.code AS country),
+  mozfun.bits28.days_since_seen(days_seen_bits) AS days_since_seen,
+  mozfun.bits28.days_since_seen(days_seen_in_tier1_country_bits) AS days_since_seen_in_tier1_country,
+  mozfun.bits28.days_since_seen(days_registered_bits) AS days_since_registered,
+  fxa.* REPLACE (cn.code AS country),
```

Comment:
this changed from `* REPLACE` to `fxa.* REPLACE`, because the view was exposing `code` and `name` columns from `cn`, which seemed wrong given the presence of `country` and `country_name`.